tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932239.yaml

```diff
@@ -387,3 +387,69 @@ tests:
             version: HTTP/1.0
           output:
             no_log_contains: id "932239"
+  - test_title: 932239-22
+    desc: "FP with word environment"
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              User-Agent: "OWASP CRS test agent"
+              Host: localhost
+              Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
+            method: GET
+            port: 80
+            uri: /get?args=environment
```

Comment:
This rule targets only `Referer` and `User-Agent` headers.

Reply:
😅